.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,8 +3,8 @@ quickly and easily:
 
  - [ ] Make sure the PR title is formatted like:
    `[BEAM-<Jira issue #>] Description of pull request`
- - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
-       Travis-CI on your fork and ensure the whole test matrix passes).
+ - [ ] Make sure tests pass via `mvn clean install -Pall-checks`. (Even better,
```

Comment:
My position is that "due diligence" does not apply to new contributors, to minimize the distance between them feeling like hacking and them becoming our friends :-)

And established contributors should know what to do anyhow. Or, realistically, I regularly use Jenkins to do this anyhow, since I rarely have fewer than 5 parallel threads of code and review going. We don't need a checkbox that duplicates the red/green from CI.

Letting people know the command to do a full run is fine, but making it a checkbox I'm not so happy with.